### PR TITLE
[wip] Guzzle request options

### PR DESCRIPTION
**Pull request by haphan**
```php
$requestOptions = [
    'verify' => false,
    'headers' => ['User-Agent' => 'PHP-OPENCLOUD/SDKv1.0']
];

$params = [
    'authUrl' => 'http://172.16.113.139:5000/v3',
    'region'  => 'RegionOne',
    'user'    =>
        array(
            'name' => 'admin',
            'password' => '034cdbdb0fd84c5f',
            'domain'   =>
                array(
                    'id' => 'default',
                ),
        ),
    'requestOptions' => $requestOptions
];


$openstack = new OpenStack\OpenStack($params);

```

### CODE REVIEW

File: src/OpenStack.php [hunk 1]
```diff
@@ -30,6 +30,9 @@ class OpenStack
      *         ['debugLog']         = (bool)              Whether to enable HTTP logging [OPTIONAL]
      *         ['logger']           = (LoggerInterface)   Must set if debugLog is true   [OPTIONAL]
      *         ['messageFormatter'] = (MessageFormatter)  Must set if debugLog is true   [OPTIONAL]
+     *         ['requestOption']    = (array)             Guzzle Http request options    [OPTIONAL]
```
Comment (jamiehannaford) [inline src/OpenStack.php]: s/requestOption/requestOptions

**Comment by jamiehannaford** [comment]
Could we also update the `php-opencloud/common` dependency to `v1.0.5` while we're at it?

**Comment by jamiehannaford** [comment]
`php-opencloud/common:v1.0.5` should now be available

**Comment by haphan** [comment]
@jamiehannaford  Please review. Thanks.

**Pull request merged**